Framework/DataHandling/src/LoadEMU.cpp

```diff
@@ -18,27 +18,31 @@
 #include "MantidKernel/UnitFactory.h"
 #include "MantidNexus/NexusClasses.h"
 
+#include <boost/filesystem.hpp>
 #include <boost/math/special_functions/round.hpp>
+#include <boost/math/tools/minima.hpp>
 
 #include <Poco/AutoPtr.h>
 #include <Poco/TemporaryFile.h>
 #include <Poco/Util/PropertyFileConfiguration.h>
 
-#include <math.h>
+#include <algorithm>
+#include <cmath>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
```

Comment:
We try to keep out `iostream` where possible. Is this required?

Reply:
No it isn't necessary, I've removed it.